dynamiq/nodes/agents/base.py

```diff
@@ -298,6 +299,8 @@ class Agent(Node):
     role: str | None = ""
     _prompt_blocks: dict[str, str] = PrivateAttr(default_factory=dict)
     _prompt_variables: dict[str, Any] = PrivateAttr(default_factory=dict)
+    _mcp_servers: list[Node] = PrivateAttr(default_factory=list)
```

Comment:
it seems this would be more clear and should work
```suggestion
    _mcp_servers: list[MCPServerAdapter] = PrivateAttr(default_factory=list)
```